README.md

```diff
@@ -110,6 +119,76 @@ docker-compose build
 ```
 docker-compose run --rm dev composer ci
 ```
+
+### Configuration Examples
+
+- AWS example (Generate Read Credentials action)
+```json
+{
+  "action": "generate-read-credentials",
+  "parameters": {
+    "backupId": null
+  },
+  "image_parameters": {
+    "storageBackendType": "s3",
+    "access_key_id": "ACCESS_KEY_ID",
+    "#secret_access_key": "SECRET_ACCESS_KEY",
+    "#bucket": "BUCKET",
+    "region": "us-east-1"
+  }
+}
+```
+
+
+- Azure example (Generate Read Credentials action)
+```json
+{
+  "action": "generate-read-credentials",
+  "parameters": {
+    "backupId": null
+  },
+  "image_parameters": {
+    "storageBackendType": "abs",
+    "accountName": "ACCOUNT_NAME",
```

Comment:
mal by si podporovat aj SAS token, ... account ma pristup ku celemu uctu, ... SAS token sa da nastavit na jeden kontajner.

Reply:
skus si vytvorit kontajner v portal.azure.com, ... a tam SAS token

Reply:
SAS token se používá potom až ke čtení v restore appce, stejně tak jako tomu je u S3

Reply:
viz: https://github.com/keboola/app-project-backup/blob/master/src/Component.php#L44

který se posílá do zálohování 
https://github.com/keboola/app-project-backup/blob/master/src/Component.php#L181

Reply:
je to tak nastavený v migrační appce která celou migraci zastřešuje:
https://github.com/keboola/app-project-migrate/blob/master/src/Migrate.php#L53

Reply:
při záloze nikde není čtení dat, tak v tom problém nevidím... v restore části ano tam by problém byl, ale k tomu už ten SAS token generujeme

Reply:
ok